### ISSUE TITLE
Fix height of subject area picker before/after selection (and other fixes)

### ISSUE THREAD

**Issue by chris-huggins** [elifesciences/elife-xpub#1333]
**Describe the bug**
The height increases when a selection is made.

**Expected behaviour**
The height would be 48px (including border) before and after selection is made (unless selectedItem(s) are more than 1 line tall - which would increase height by 48px for each line/item.

Also:

- Update text and styling of the placeholder text (should match other inputs)
- Remove dropdown icon/arrow
- Fix selectedItem height

**Current**

![image](https://user-images.githubusercontent.com/41907215/51385922-92594600-1b18-11e9-82e4-ae2a2562669c.png)

**Intended**

![image](https://user-images.githubusercontent.com/41907215/51385824-40b0bb80-1b18-11e9-99f5-9f9a21c18c68.png)

Select Item - should be 36px tall, 5px padding within the input, and 6px between selectedItems

![image](https://user-images.githubusercontent.com/41907215/51385907-82d9fd00-1b18-11e9-988f-6576a8225024.png)

**Comment by chris-huggins** [comment]
Also,

We should clear the placeholder text (which should not be black) when the input has focus so that it is clearer the users that they can type to search/filter.

Currently the placeholder text "Select..." persists until selection even though the user is able to type.

![image](https://user-images.githubusercontent.com/41907215/51395645-2802ce80-1b35-11e9-9270-3a5fb99438d4.png)

**Comment by chris-huggins** [comment]
We should also remove the validation state while it has focus (image shown above has both blue focus border and red validation border

**Comment by SemraSh** [comment]
Current view
![image](https://user-images.githubusercontent.com/32770804/51539498-1464aa00-1e4c-11e9-93c4-b6c8682e3f1c.png)
as @chugginselifesciences approved the required changes are as follows
![image 1](https://user-images.githubusercontent.com/32770804/51539867-f8153d00-1e4c-11e9-939f-4614326d85f3.png)
- The Article type field should be 4 columns instead of 8. 
- The placeholders are supposed to be colorText
- The arrow on the Subject areas type will be removed
- The arrow on the Article type will be replaced with a new icon
- The Subject areas title will be replaced with Subject area(s)
- The size of the input field will be updated to be 48px at all times ( see above )


> Additional notes
Those input components are build with different libraries
The Subject areas input ( SubjectAreaDropdown ) is a component that we've created and uses  a library called `react-select`.
The Article type is using a component called `Menu` and that component is in pubsweet but we've added our styles in client/elife-theme/src/elements/ui/Menu.js
I would suggest creating a new higher order component for both of them sharing similar base styles and changing properties like `arrowIcon={true} width={4}` etc. Or using Menu component for both of them and changing the current styles accordingly.
